### DISCUSSION TITLE
Hacktoberfest accepted marked as EXCLUDED on dashboard?

### DISCUSSION THREAD

**Discussion by CelestialProgrammer** [community/community#69083] [New to GitHub]
Why earlier there was waiting time on my accepted PR now it is showing excluded?

**Comment by anasjawed283** [comment]
The Repo that you have contributed on must not have followed hacktoberfest guidelines and is removed from Hacktoberfest, PRs on that will not be considered as contribution.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
